apps/src/lab2/views/dialogs/DialogManager.tsx

```diff
@@ -83,6 +116,13 @@ const DialogManager: React.FunctionComponent<DialogManagerProps> = ({
 
 interface DialogControl {
   showDialog: (dialogType: DialogType, callback: () => void) => void;
+  showGenericDialog: (
+    dialogType: DialogType,
+    callback: () => void,
+    title: string,
+    message: string,
+    confirmText?: string
```

Comment:
Thoughts on just adding these as optional params to `showDialog` instead of having a separate function? I can imagine these being useful in other cases as well (for example in the Start Over dialog where we currently read the app type to determine what specific message to show, maybe instead labs can just pass in a string to use)

Reply:
That seems fine to me, will update